package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "mozdevice",
+  "version": "1.0.0",
+  "description": "Interactions for Mozilla devices and environments",
+  "main": "index.js",
+  "author": "Eli Perelman",
+  "license": "MIT",
+  "keywords": [
+    "firefox-os",
+    "b2g"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/eliperelman/node-mozdevice.git"
+  },
+  "dependencies": {
+    "adbkit-logcat": "^1.0.3",
+    "async": "^0.9.0",
+    "debug": "^2.1.1",
+    "promise": "^6.0.1"
```

Comment:
I guess "child_process" is a built-in module and not an npm package? (I didn't see it on the npm site).